packages/typespec-client-generator-core/src/context.ts

```diff
@@ -38,12 +38,11 @@ export interface TCGCEmitterOptions extends SdkEmitterOptions {
   "emitter-name"?: string;
 }
 
-export interface SdkEmitterOptions {
+export interface UnbrandedSdkEmitterOptions {
```

Comment:
will all the emitters supprot those or it will still be pick an dchoose, might be beter to just export each one separately if thats the case

Reply:
i agree. e.g., java does not want any flag for unbranded.